controller/volume_controller.go

```diff
@@ -880,6 +881,9 @@ func (vc *VolumeController) createCronJob(v *longhorn.Volume, job *types.Recurri
 	}
 	if job.Type == types.RecurringJobTypeBackup {
 		cmd = append(cmd, "--backuptarget", backupTarget)
+		if backupTarget != "" && strings.Contains(backupTarget, "s3") {
```

Comment:
Don't show keys here in the cmdline. It will be shown in the CronJob spec. Use secret and mount the secret instead.